metadata.json

```diff
@@ -1,17 +1,16 @@
 {
   "name": "puppet-augeasproviders_sysctl",
-  "version": "2.6.2",
+  "version": "3.0.0-rc0",
   "author": "Raphael Pinson, Trevor Vaughan",
   "summary": "Augeas-based sysctl type and provider for Puppet",
   "license": "Apache-2.0",
   "source": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
   "project_page": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
   "issues_url": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl/issues",
-  "description": "This module provides type/provider for sysctl using the Augeas configuration API library.",
   "dependencies": [
     {
       "name": "puppet/augeasproviders_core",
-      "version_requirement": ">= 2.4.0 < 4.0.0"
+      "version_requirement": ">= 2.4.0 < 5.0.0"
```

Comment:
I decided to bump this dependency so that removing EoL operating systems in core is covered.